core/src/main/java/com/datastax/oss/driver/api/core/data/CqlVector.java

```diff
@@ -56,36 +62,48 @@ public int hashCode() {
 
   @Override
   public String toString() {
+    StringBuilder builder = new StringBuilder("CqlVector{");
+    for (T value : values) {
+      builder.append(value).append(", ");
+    }
+    builder.setLength(builder.length() - ", ".length());
+    builder.append("}");
+    return builder.toString();
+  }
```

Comment:
Note that the TypeCodec interface provides methods for converting a specific type to a String and back.  Upshot is that whatever is used here has to play nice with what's in [CqlVectorCodec](https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/oss/driver/internal/core/type/codec/CqlVectorCodec.java#L118-L134).

CqlDurationCodec provides [a useful example](https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/oss/driver/internal/core/type/codec/CqlDurationCodec.java#L103-L115) of the interaction here.

Reply:
noted